howso/details_residuals.amlg

```diff
@@ -485,77 +485,78 @@
 		;pull the appropriate robust or standard residuals
 		(assign (assoc global_feature_residuals (get !residualsMap param_path)))
 
+		;if global_feature_residuals is empty, we call react_aggregate to cache the required residuals
 		(if (= 0 (size global_feature_residuals))
 			(let
 				(assoc hp_param_path (get hyperparam_map "paramPath"))
-				;output a detailed warning why global convictions aren't computed
-				(accum (assoc
-					warnings
-						(associate (concat
-							"Cannot compute 'global_case_feature_residual_convictions_full' because Full"
-							" global residuals have not been computed for action_feature "
-							(if (size action_features)
-								(concat "'" (first hp_param_path) "'")
-								"'.targetless'"
+				(let
+					(assoc
```

Comment:
I suggest getting rid of this entire let-assoc and inline these values directly into the react_aggregate call as parameters since a) these variables aren't going to be reused below b) they are all simple one-liners